### ISSUE TITLE
How to pass checked data from dynamic table to controller post method 

### ISSUE THREAD

**Issue by amit018-hub** [smartstore/SmartStoreNET#2262]
@{
    Layout = "~/Views/Shared/_AdminLayout.cshtml";
}

<div class="row">
    <div class="col-xl-12">
        <div id="panel-1" class="panel">
            <div class="panel-container show">
                <div class="panel-content">
                    <div class="row mb-1">
                        <div class="col-md-4">
                            <div class="form-group">
                                <label>Employee Name:</label>
                                @*<input type="hidden" name="EmpId" class="form-control" id="EmpId"/>*@
                                <input type="text" class="form-control" id="Emp_Name" name="Emp_Name" />
                            </div>
                        </div>
                        <div class="col-md-4">
                            <div class="form-group">
                                <label>Employee Branch:</label>
                                <input type="text" class="form-control" id="Emp_Branch" name="Emp_Branch" />
                            </div>
                        </div>
                        <div class="col-md-4">
                            <div class="form-group">
                                <label>Employee Salary:</label>
                                <input type="text" class="form-control" id="Emp_Salary" name="Emp_Salary" />
                            </div>
                        </div>
                    </div>
                    <div class="row mb-2">
                        <div class="col-md-10">

                        </div>
                        <div class="col-md-2">
                            <button type="button" class="btn btn-outline-primary" id="btn-add" style="float:right; margin-top:0px;">Add More</button>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>

<form action="/Home/CreateEmp" method="post">
    <div class="card" id="cardtable">
        <div class="card-body">
            <h6 class="mb-0 text-uppercase">Employee Details</h6>
            <hr>
            <div class="row mb-3">
                <div class="col-md-12">
                    <table class="table table-bordered " id="scrap">
                        <thead>
                            <tr class="card_header">
                                @*text-light*@
                                <th>Check</th>
                                <th>Employee Name</th>
                                <th>Employee branch</th>
                                <th>Employee Salary</th>
                            </tr>
                        </thead>
                        <tbody id="tb">
                        </tbody>
                    </table>

                </div>
            </div>

            <div class="row mb-3">
                <div class="col-md-10"></div>
                <div class="col-md-2">
                    <button type="submit" id="btnsumit" class="btn btn-outline-success">Submit</button>
                    <button type="button" class="btn btn-warning waves-effect waves-themed" style="float:right;" onclick="location.href='@Url.Action("CreateEmp","Home")';">Cancel</button>
                </div>
            </div>
            <p>Total Number Of Employees:<label></label></p>
            <p>Total Salary:<label></label></p>
        </div>
    </div>
</form>

<script src="~/Scripts/jquery-3.4.1.min.js"></script>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
<script>

    

    var count = 0;
    $(function () {
        $('#btn-add').click(function () {
            debugger;
            var rows = "";
            count = count + 1;
            var empname = $('#Emp_Name').val();
            var empbranch = $('#Emp_Branch').val();
            var empsalary = $('#Emp_Salary').val();
            if (empname != "" && empbranch != "" && empsalary != "") {
                rows += '<tr><td><input type="checkbox" id="secheck" /><td><input type="hidden" name="Emp_Name" value="' + empname + '" />' + empname + '</td><td><input type="hidden" name="Emp_Branch" value="' + empbranch + '" />' + empbranch + '</td><td><input type="hidden" name="Emp_Salary" value="' + empsalary + '" />' + empsalary + '</td></td><tr>';
                $(rows).appendTo("tbody");
                $('#Emp_Name').val("");
                $('#Emp_Branch').val("");
                $('#Emp_Salary').val("");
                //var allVals = [empname, empbranch, empsalary];
                //$('#secheck :checked').each(function () {
                //    allVals.push($(this).val());
                //});
            }
            else {
                alert("Field Empty");
            }
        });
    });

    $(function () {
        
        $("#btnsumit").click(function () {
            debugger;
            var selected = new Array();

            $("#tb input[type=checkbox]:checked").each(function () {
                selected.push(this.value);
            });

            if (selected.length > 0) {
                alert("Selected values: " + selected.join(","));
            }
        });
    });

</script>
